src/axolotl/loaders/tokenizer.py

```diff
@@ -295,4 +295,8 @@ def _load_mistral_common_tokenizer(cfg: DictDefault):
         LOG.info(
             "No Chat template selected. Consider adding a chat template for easier inference."
         )
+
+    # make the tokenizer.pad call quieter 🤐
+    tokenizer.deprecation_warnings["Asking-to-pad-a-fast-tokenizer"] = True
```

Comment:
Can we do, `hasattr(tokenizer, "deprecation_warnings")` else it'll break for mistral tokenizer